src/mono/mono/metadata/debug-mono-ppdb.c

```diff
@@ -442,7 +446,14 @@ mono_ppdb_lookup_location (MonoDebugMethodInfo *minfo, uint32_t offset)
 MonoImage *
 mono_ppdb_get_image (MonoPPDBFile *ppdb)
 {
-    return  ppdb->image;
+    return ppdb->image;
```

Comment:
I think the diff here is an indentation change (tabs->spaces)? The changes here as a whole seems to use spaces :(

Reply:
Fixed.